hack/install-buildx

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env bash
 set -eu -o pipefail
-topdir="$(realpath $(dirname "$0")/..)"
-cd "${topdir}"
 
-VERSION="v0.5.1"
-BINDIR="$(pwd)/bin"
-DEST="${BINDIR}/buildx"
+if [ -z "${BINDIR}" ] && [ -z "${PREFIX}" ]; then
+  PREFIX="$(realpath "$(dirname "$0")"/..)"
```

Comment:
fwiw, this needs some work if we want it to be functional on macOS; see https://stackoverflow.com/questions/3572030/bash-script-absolute-path-with-os-x